README.md

```diff
@@ -47,6 +47,21 @@ To ensure your markdown is properly formatted, run `make markdowntest`.
 
 To check for broken links run `make htmltest`.
 
+## Linters and Checks
+
+To ensure that you do not accidentally add `EXIF` data on images, please add the following lines to your `.git/hooks/pre-commit` file:
```

Comment:
```suggestion
To ensure that you do not accidentally add `EXIF` data on images, please add the following lines to your `.git/hooks/pre-commit` file. If you do not have a file of that name in that location, create one, or rename the existing `pre-commit.sample` file in that directory as `pre-commit`, and add the code below:
```

Reply:
Worth noting that `.git` is explicitly excluded in our git repo config, so this file is not part of our version control workflow. i.e. You gotta add this to your local clone of your fork by hand.

Reply:
It's not really explicitly excluded. You can't manage the .git folder in git in any project. Happy to add clarification though.